platform/ios/jazzy.yml

```diff
@@ -23,53 +26,61 @@ custom_categories:
       - MGLMapViewDelegate
       - MGLStyle
       - MGLUserTrackingMode
-  - name: Annotations
+  - name: User Interaction
```

Comment:
Now that this section only contains callout views, I think we should move it farther down the list — perhaps after the new Annotations section?